stories/Collapse/CollapseNestedExample.js

```diff
@@ -1,109 +1,88 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { Accordion, AccordionHeader, AccordionBody } from '../../src'
 
-class CollapseNestedExample extends React.Component {
-  state = {
-    collapseOpen1: true,
-    collapseOpen2: false,
-    collapseOpen3: false
-  }
+const CollapseNestedExample = () => {
+  const [collapseOpen1, setCollapseOpen1] = useState(true)
+  const [collapseOpen2, setCollapseOpen2] = useState(false)
+  const [collapseOpen3, setCollapseOpen3] = useState(false)
+  return (
+    <Accordion>
+      <AccordionHeader
+        active={collapseOpen1}
+        onToggle={() => setCollapseOpen1(!collapseOpen1)}>
+        Collapse header #1
+      </AccordionHeader>
+      <AccordionBody active={collapseOpen1}>
+        <CollapseNestedPanel />
```

Comment:
This component is not shown in the Source Code panel. Can you move its state and JSX here? It will become longer but it's okay I guess for explanatory reasons.

Reply:
Yes, fair enough.